File: TemplateMethod/CreaturesCardGame/Creature.cs
namespace CreaturesCardGame
{
    public class Creature
    {
        public string Name;
        public int Health;
        public int Attack;

        public Creature(string name, int attack, int health)
        {
            Name = name;
            Attack = attack;
            Health = health;
        }
    }
}

File: ChainOfResponsibility/Python/MethodChain/creature.py
class Creature:
    def __init__(self, name: str, attack: int, defense: int) -> None:
        self.name = name
        self.attack = attack
        self.defense = defense

    def __str__(self) -> str:
        return f"{self.name} ({self.attack}/{self.defense})."
